user/lxc/patches/meson.patch

```diff
@@ -0,0 +1,23 @@
+--- lxc-5.0.1/meson.build	2022-07-27 21:20:41.000000000 -0700
++++ lxc-5.0.1.bk/meson.build	2024-08-24 09:19:18.731887904 -0700
+@@ -316,7 +316,8 @@
+ if time_epoch == ''
+     time_epoch = run_command(date, '+%s', check: true).stdout()
+ endif
+-generate_date = run_command(date, '--utc', '--date=@' + time_epoch, '+%Y-%m-%d', check: true).stdout().strip()
++#generate_date = run_command(date, '--utc', '--date=@' + time_epoch, '+%Y-%m-%d', check: true).stdout().strip()
++generate_date = run_command(date, '+%Y-%m-%d', check: true).stdout().strip()
```

Comment:
this doesn't seem correct since the previous code uses --utc and stuff. i dunno how would would inject source_date_epoch into it so i'll let someone else think about it

Reply:
Both commands come up with the same result.

Reply:
only if /etc/localtime happens to be utc. for bsd date, `-z utc` is equivalent

Reply:
fixed